pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -1275,54 +1340,93 @@ func componentResourceRemoval(o interface{}, c client.Client, cr *hcov1alpha1.Hy
 	return err
 }
 
-func ensureCDIDeleted(c client.Client, instance *hcov1alpha1.HyperConverged) error {
+func ensureKubeVirtDeleted(c client.Client, instance *hcov1alpha1.HyperConverged, requestNamespace string) error {
+	virt := newKubeVirtForCR(instance, requestNamespace)
```

Comment:
isn't this just `instance.Namespace`?

Reply:
formally not, but yes, now we are 100% sure that it's the same value by design (being a real singleton).
Let's simplify this one.